views/component/recipients.js

```diff
@@ -0,0 +1,82 @@
+const { h, map } = require('mutant')
+const addSuggest = require('suggest-box')
+
+function Recipients (opts) {
+  const {
+    state,
+    myKey,
+    suggest = {},
+    avatar = (id, size) => id,
+    i18n = (key) => {
+      const translations = { 'gathering.action.addMoreRecps': '(optional) invite to this scry' }
+      return translations[key]
+    }
+  } = opts
+
+  return h('GatheringRecipients', [
```

Comment:
we can't leave this as GatheringRecipients, otherwise it will collide with the styles of Gatherings ... 
Make this ScryRecipients

Reply:
oh yes! changed.